### PR TITLE
LPS-37843 SF - no need to contact server.

### PR DESCRIPTION
**Pull request by brandizzi**
# _CHALLENGE!_

### CODE REVIEW

File: modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/scheduler.js [hunk 1]
```diff
@@ -438,19 +438,18 @@ AUI.add(
 							);
 						}
 						else if (schedulerEvent.isMasterBooking()) {
-							remoteServices.getCalendarBookingInvitees(
```
Comment (brandizzi) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/scheduler.js]: # _CHALLENGE!_

**Comment by marcellustavares** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by marcellustavares** [comment]
hahahah @brandizzi!

**Comment by marcellustavares** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/44817. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**